### DISCUSSION TITLE
Remove symbols from tables​

### DISCUSSION THREAD

**Discussion by utilisateuralpha** [toeverything/blocksuite#6680] [Ideas]
### Description

I propose to remove the T symbols in front of the main property cells in the tables. This does not make it easier to read the contents of the tables.
It's bulky and useless (especially in kanban mode).
The other symbols also in kanban mode make reading difficult.
See the following two images:

![symboleT2](https://github.com/toeverything/AFFiNE/assets/89381501/ab951e79-d8a9-4381-98d7-cbff9f1b4eac)
![symboleT1](https://github.com/toeverything/AFFiNE/assets/89381501/059e230c-a984-42da-af43-8c096f8838e1)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by zanwei** [comment]
hi @utilisateuralpha 

The reason why there are type icons here (the ones you selected on the image) is because markdown in the database does not support displaying styles (like heading 1-6 and quote, etc), so we added icons to help users understand the style of the current content. What do you think?

**Comment by utilisateuralpha** [comment]
Hi @zanwei 

Thanks for your reply. It's not about the styles (h1, h2, ...) but the type of properties (text, number, ...).
Here are two cases to discuss:

1) With icons: in kanban display mode, the T symbol (corresponding to the main column) must be displayed before the cell contents and without a line break (like the other columns).

2) Without icons: I think this is much better. The user doesn't need to know the type of properties defined. They only need to read the content (notion.so, for example, hides the display of property types). If you prefer to keep these icons, then please add an option to hide them.

3) With and without icons: in kanban mode, the line surrounding the entire contents of a table row should be wider (or of a different color) than the line separating the contents of the main columns (T icon) from the other columns.

That's my opinion, what do you think?

**Comment by zanwei** [comment]
Hi @utilisateuralpha 

Glad to hear your feedback and will consider that. Actually we were planning to modify database soon, please stay tuned.
